src/Symfony/Component/Messenger/Transport/Doctrine/Connection.php

```diff
@@ -0,0 +1,224 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Messenger\Transport\Doctrine;
+
+use Doctrine\DBAL\Connection as DriverConnection;
+use Doctrine\DBAL\Exception\TableNotFoundException;
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\DBAL\Schema\Synchronizer\SingleDatabaseSynchronizer;
+use Doctrine\DBAL\Types\Type;
+use Symfony\Component\Messenger\Exception\InvalidArgumentException;
+
+/**
+ * @author Vincent Touzet <vincent.touzet@gmail.com>
+ */
+class Connection
+{
+    /**
+     * Configuration of the connection.
+     *
+     * * table_name : name of the table
+     * * manager : name of the Doctrine's entity manager
+     * * redeliver_timeout : Timeout before redeliver messages still in handling state (i.e: available_at is not null and message is still in table). Default 3600
```

Comment:
I don't think the `available_at` part is correct, right? It's really all about `delivered_at` I think - if the message has a `delivered_at`... but still exists in the table... then 3600 seconds later, we will try it again. (Btw, the feature itself worked AWESOME)

Reply:
Yes it' all about delivered_at. Typo in the doc comment ^^